pkg/statistics/handle/bootstrap.go

```diff
@@ -40,6 +40,8 @@ import (
 	"go.uber.org/zap"
 )
 
+const initStatsStep = int64(500)
```

Comment:
Why was this value changed from the default 1000 to 500. What is the impact of changing the size?

Reply:
I'm concerned that having a partitioned table within a single task might result in a very slow load time.